### DISCUSSION TITLE
Does WinUI provide a Generic Picker control ?

### DISCUSSION THREAD

**Discussion by jeromelaban** [unoplatform/uno#10107] [Q&A]
[_From Discord:_](https://discord.com/channels/372137812037730304/550416151172087808/1029197257968455690)

Hi all, is in Uno available a generic Picker like in Xamarin.Forms or similar one? https://www.c-sharpcorner.com/article/picker-control-in-xamarin-forms-application-for-android-and-uwp/

**Comment by jeromelaban** [comment]
The closest control in WinUI is [`ComboBox`](https://playground.platform.uno/#combobox). Uno provides platform specific implementations that use native APIs for items selection, which you can apply by [changing the ComboBox style like this](https://github.dev/unoplatform/calculator/blob/a7b61e8e1dc8bb4908f7711740c9ad60053ffb15/src/Calculator.Shared/Views/UnitConverter.xaml#L49-L52), using the [native iOS Picker API](https://github.com/unoplatform/uno/blob/d9a87b3b07a9fd90c9a12d276a406a495dd1fd48/src/Uno.UI/UI/Xaml/Controls/Picker/Picker.iOS.cs#L28).

**Reply by luciusinfabula** [reply]
Hi Jerome, thanks for your kindly answer.
Luc
